src/EnergyPlus/SizingManager.cc

```diff
@@ -1432,13 +1417,8 @@ namespace SizingManager {
 							ZoneSizingInput( ZoneSizIndex ).ZoneName = "Invalid Zone Name";
 						}
 					}
-					IsNotOK = false;
-					IsBlank = false;
-					InputProcessor::VerifyName( ZoneSizingInput( ZoneSizIndex ).ZoneName, ZoneSizingInput, &ZoneSizingInputData::ZoneName, ZoneSizIndex - 1, IsNotOK, IsBlank, cCurrentModuleObject + " Name" );
-					if ( IsNotOK ) {
-						ErrorsFound = true;
-						if ( IsBlank ) cAlphaArgs( 1 ) = "xxxxx";
-					}
+
+					InputProcessor::IsNameEmpty( cAlphaArgs(1), cCurrentModuleObject + " Name", "xxxxx", ErrorsFound);
```

Comment:
This one looks odd. Just double check that it is only checking within the same object and we can remove the VerifyName call and don't need `VerifyUniqueInterObjectName`